Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageFullScreenClient.cpp

```diff
@@ -61,6 +61,13 @@ void InjectedBundlePageFullScreenClient::enterFullScreenForElement(WebPage* page
         page->send(Messages::WebFullScreenManagerProxy::EnterFullScreen(blocksReturnToFullscreenFromPictureInPicture, WTFMove(mediaDetails)));
 }
 
+#if ENABLE(QUICKLOOK_FULLSCREEN)
+void InjectedBundlePageFullScreenClient::updateImageSource(WebPage* page, FullScreenMediaDetails&& mediaDetails)
```

Comment:
This doesn't belong in `InjectedBundlePageFullScreenClient` since you aren't actually interacting with the injected bundle. You can just send this message directly in `WebFullScreenManager::updateImageSource`.